spec/concepts/todo/create_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 RSpec.describe Todo::Create do
   it "can be spec'd" do
-    raise "Implement me"
+    todo = Todo::Create.(todo: {title: "Sample title"}).model
+    expect(todo.title).to eq("Sample title")
   end
```

Comment:
Hi sir,

1. You mean title sir? If so, if the title is specified, like for example is "Sample title", the RSpec expects to see the title of the todo to be "Sample title".
2.  If there is no title passed or the title is not equivalent to "Sample title", but the RSpec is still expecting the title, then the test will fail.